duckduckgo/Android
# Support for Import / Export Favorites

Task/Issue URL: https://app.asana.com/0/488551667048375/1200392101370979

**Description**:
This PR adds support for Importing / Exporting Favorites. When we export our Favorites we name the folder "DuckDuckGo Favorites", so we look for that folder name when traversing an exported file. Anything which falls under another folder will be added to the unique "Bookmarks" folder.

**Steps to test this PR**:
1. Open app
2. Adds Favorites and Bookmarks
3. Export them
4. Delete all Favorites and Bookmarks
5. Import the previously exported file
6. The items should be the same as before deleting them.

Demo: https://user-images.githubusercontent.com/531613/123437311-d4cbba80-d5cf-11eb-9900-9f13539ad50b.mp4

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
